spyder/widgets/collectionseditor.py

```diff
@@ -1132,57 +1133,41 @@ def view_item(self):
-                return True
-            except Exception:
-                QMessageBox.warning(self, _("Import error"),
-                                    _("Please install <b>matplotlib</b>"
-                                      " or <b>guiqwt</b>."))
-
     def plot_item(self, funcname):
         """Plot item"""
         index = self.currentIndex()
-        if self.__prepare_plot():
-            if self.proxy_model:
-                key = self.source_model.get_key(
-                    self.proxy_model.mapToSource(index))
-            else:
-                key = self.source_model.get_key(index)
-            try:
-                self.plot(key, funcname)
-            except (ValueError, TypeError) as error:
-                QMessageBox.critical(self, _( "Plot"),
-                                     _("<b>Unable to plot data.</b>"
-                                       "<br><br>Error message:<br>%s"
-                                       ) % str(error))
+        if self.proxy_model:
+            key = self.source_model.get_key(
+                self.proxy_model.mapToSource(index))
+        else:
+            key = self.source_model.get_key(index)
+        try:
+            self.plot(key, funcname)
+        except (ValueError, TypeError) as error:
+            QMessageBox.critical(self, _( "Plot"),
+                                    _("<b>Unable to plot data.</b>"
+                                    "<br><br>Error message:<br>%s"
+                                    ) % str(error))
 
     @Slot()
     def imshow_item(self):
         """Imshow item"""
         index = self.currentIndex()
-        if self.__prepare_plot():
-            if self.proxy_model:
-                key = self.source_model.get_key(
-                    self.proxy_model.mapToSource(index))
+        if self.proxy_model:
+            key = self.source_model.get_key(
+                self.proxy_model.mapToSource(index))
+        else:
+            key = self.source_model.get_key(index)
+        try:
+            if self.is_image(key):
+                self.show_image(key)
             else:
-                key = self.source_model.get_key(index)
-            try:
-                if self.is_image(key):
-                    self.show_image(key)
-                else:
-                    self.imshow(key)
-            except (ValueError, TypeError) as error:
-                QMessageBox.critical(self, _( "Plot"),
-                                     _("<b>Unable to show image.</b>"
-                                       "<br><br>Error message:<br>%s"
-                                       ) % str(error))
+                self.imshow(key)
+        except (ValueError, TypeError) as error:
+            QMessageBox.critical(self, _( "Plot"),
+                                    _("<b>Unable to show image.</b>"
+                                    "<br><br>Error message:<br>%s"
+                                    ) % str(error))
```

Comment:
```suggestion
            QMessageBox.critical(
                self,
                _( "Plot"),
                _("<b>Unable to show image.</b><br><br>"
                  "The error message was:<br>%s") % str(error)
            )
```

Improve formatting and message a bit.

Reply:
See my comment on [Black](https://black.readthedocs.io/en/stable/) and [isort](https://pycqa.github.io/isort/).